src/plots/subplot_defaults.js

```diff
@@ -0,0 +1,73 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var Lib = require('../lib');
+var Plots = require('./plots');
```

Comment:
non-:no_entry_sign:, but I think we should make a new directory called something like `src/plots/base/` where shared subplot files would resides. 

We should try to make our subdirectory structures consistent. Thoughts?